imports/ui/views/users/UserPage.jsx

```diff
@@ -1,6 +1,7 @@
 import React, {Component} from 'react'
 
 import VolunteerExperienceList from '/imports/ui/views/users/volunteerExperienceList/VolunteerExperienceList'
+import ProfessionalExperience from '/imports/ui/views/users/professionalExperience/ProfessionalExperience'
```

Comment:
I would recommend creating an index.js file inside the professionalExperience directory and renaming that directory to ProfessionalExperience. Refer to either the VolunteerExperienceList or UsersList directory in the current December branch for creating an index.js file